Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/task/data/model/Id.java

```diff
@@ -3,7 +3,7 @@
 import com.google.auto.value.AutoValue;
 
 @AutoValue
-public abstract class Id {
+public abstract class Id implements Comparable<Id> {
```

Comment:
The data is ordered as a mean to ensure consistency during concurrent modification.